### DISCUSSION TITLE
Playbooks alert suppress for specific IP

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#7834] [Unsupported Versions] [answered]
Hi friends,
             I have made all playbooks active, now I want o mature the service according to my infrastructure
First I want to suppress alerts generate from specific IP 
Anyone have Idea how to do for Playbook?

**Comment by defensivedepth** [comment, marked as answer]
Playbook Plays are based off of Sigma - you will need to edit the Sigma of the Play to account for local configurations that generate false positives.

For instance, in one of my SO environments, I am seeing a large amount of Non Interactive PowerShell alerts. Drilling down into the alerts, it appears to be a legitimate execution of CompatTelRunner.exe. I went ahead and filtered this out by:

    Copying the Sigma from the Play (found under the Sigma field) and pasting it into the left pane under "Create New Play"

    I then clicked on "Convert" to make sure that it converted correctly

    Then added CompatTelRunner.exe under the filter clause and Converted it again to make sure it worked ([screencap](https://www.screencast.com/t/SShb9dszZ))

    Then I copy and pasted the edited sigma back to the Play under the Sigma field (drop it in between <pre><code class="yaml"> and </code></pre> tags

    Finally, clicked "Submit" - Playbook will take care of the rest.

You can edit the Sigma right there in the Sigma field in the Play, but it is not a YAML editor and sometimes it is easier to edit using a YAML editor.

One caveat - if there is ever an update for that Sigma rule from the Sigma rules repo, your changes will get overwritten.

Finally - if you are seeing legitimate executions that are not unique to your environment, you might consider submitting a PR to the rule in the Sigma repo. (https://github.com/SigmaHQ/sigma/tree/master/rules)
